pkg/assetfetcher/http.go

```diff
@@ -183,7 +189,13 @@ func (c *NexposeAssetFetcher) newNexposeSiteAssetsRequest(siteID string, page in
 func (a Asset) hasBeenScanned() bool {
 	for _, evt := range a.History {
 		if evt.Type == "SCAN" {
-			return true
+			t, err := time.Parse(time.RFC3339, evt.Date)
+			// check if the date field is parsable and if it isn't 0 value
+			if err != nil || t.IsZero() {
+				continue
+			} else {
```

Comment:
Suggestion: `else` is not really considered idiomatic in go - https://golang.org/doc/effective_go.html#if. Since you have a continue on line 195, you can drop the else and just have a bare `return true` after the if block.

Reply:
done!